code/addons/themes/docs/getting-started/emotion.md

```diff
@@ -0,0 +1,77 @@
+# 🏁 Getting started with `@emotion/styled`
+
+## 📦 Install addon
+
+To get started, **install the package** as a dev dependency
+
+yarn:
+
+```zsh
+yarn add -D @storybook/addon-themes
+```
+
+npm:
+
+```zsh
+npm install -D @storybook/addon-themes
+```
+
+pnpm:
+
+```zsh
+pnpm add -D @storybook/addon-themes
+```
+
+## 🧩 Register Addon
+
+Now, **include the addon** in your `.storybook/main.js` file
+
+```diff
+module.exports = {
+  stories: [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-essentials",
++   "@storybook/addon-themes"
+  ],
+};
+```
+
+## 🎨 Provide your theme(s)
+
+Finally, provide your theme(s) and global styles component to your stories with our `withThemeFromJSXProvider` decorator.
+
+Make the following changes to your `.storybook/preview.js`
+
+```diff
++import { withThemeFromJSXProvider } from "@storybook/addon-themes";
++import { ThemeProvider } from '@emotion/react';
++import { GlobalStyles, lightTheme, darkTheme } from "../src/themes"; // import your custom theme configs
+
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    expanded: true,
+    hideNoControlsWarning: true,
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+};
+
++export const decorators = [
++  withThemeFromJSXProvider({
```

Comment:
The current names are more descriptive than what you're suggesting, what led you all to the new names? Also, this would be a bigger change than just changing the package it's exported from which is nicer from a change management perspective.

Reply:
It's about subjective preference, I would say. So stay with the naming if you feel strongly about it.